deploy.sh

```diff
@@ -30,4 +27,5 @@ else
     ssh-add deploy_key
     echo "$SERVER $SERVER_PUBLIC_KEY" > known_hosts
     rsync --archive --chmod="D755,F644" --verbose --compress --delete --rsh="ssh -o UserKnownHostsFile=known_hosts" ./whatwg.org ./*.whatwg.org "deploy@$SERVER:/var/www/"
+    echo ""
```

Comment:
I'm surprised this is necessary/helpful?

Reply:
I'm not actually sure, I'll remove it for now.